src/patient-registration/input/input.css

```diff
@@ -82,3 +82,35 @@
 .identifierInput {
     height: 80px !important;
 }
+
+.searchInput {
+  width: 660px !important;
```

Comment:
Mobile?
660px sounds like it would exceed most mobile screens (~480px)

Reply:
I changed it to be the same length as the other input fields in this section for now. The appearance might be touched again anyway when the autocomplete gets carbonised?